javascript/ql/lib/semmle/javascript/security/dataflow/StoredXssCustomizations.qll

```diff
@@ -5,6 +5,11 @@
 
 import javascript
 
+/**
+ * Provides sources, sinks, and sanitizers for reasoning about
+ * cross-site scripting vulnerabilities where the taint-flow passes through a thrown
+ * exception.
+ */
```

Comment:
```suggestion
/**
 * Provides default sources, sinks and sanitizers for reasoning about
 * stored cross-site scripting vulnerabilities.
 */
```
Mentions a "thrown exception". I've just suggested the same QLDoc from the file.

Reply:
Right. I accidentally copy-pasted from the wrong place.